diff --git a/node_modules/miniflare/dist/src/index.js b/node_modules/miniflare/dist/src/index.js
--- a/node_modules/miniflare/dist/src/index.js
+++ b/node_modules/miniflare/dist/src/index.js
@@ -72524,7 +72524,7 @@
     if (!options.assets) {
       return [];
     }
-    const storageServiceName = `${ASSETS_PLUGIN_NAME}:storage`;
+    const storageServiceName = `${ASSETS_PLUGIN_NAME}:storage:${options.assets.workerName}`;
     const storageService = {
       name: storageServiceName,
       disk: {
